process/eventsInterceptor.go

```diff
@@ -88,19 +79,16 @@ func (ei *eventsInterceptor) ProcessBlockEvents(eventsData *data.ArgsSaveBlockDa
 	}, nil
 }
 
-func (ei *eventsInterceptor) getLogEventsFromTransactionsPool(logs []*data.LogData) []data.Event {
+func (ei *eventsInterceptor) getLogEventsFromTransactionsPool(logs []*outport.LogData) []data.Event {
 	var logEvents []*logEvent
 	for _, logData := range logs {
 		if logData == nil {
 			continue
 		}
-		if check.IfNil(logData.LogHandler) {
-			continue
-		}
 
-		for _, eventHandler := range logData.LogHandler.GetLogEvents() {
+		for _, event := range logData.Log.Events {
```

Comment:
should we consider a nil check on `logData.Log` here ? :thinking:

Reply:
added nil check